src/jailer/src/cgroup.rs

```diff
@@ -217,9 +217,7 @@ impl Cgroup {
 
             // And now add "tasks" to get the path of the corresponding tasks file.
             path_buf.push("tasks");
-            if !tasks_files.contains(&path_buf) {
```

Comment:
This check is needed. Multiple controllers can reside on the same `cgroup` mount path.

For example on my local system:
```
~$ grep cgroup /proc/mounts | grep -w "cpuacct\|cpu"
cgroup /sys/fs/cgroup/cpu,cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpu,cpuacct 0 0
```

Both the `cpu` and `cpuacct` controllers are mounted in the same `/sys/fs/cgroup/cpu,cpuacct` path.

Reply:
Tnx acatan 😃 , I agree this. actually I missed this case.